BookStore-Angular-MongoDb/angular/package.json

```diff
@@ -57,7 +58,8 @@
     "karma-coverage": "~2.1.0",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.7.0",
-    "ng-packagr": "^18.1.0",
-    "typescript": "~5.4.0"
-  }
-}
\ No newline at end of file
+    "ng-packagr": "~20.0.0",
+    "typescript": "~5.8.0"
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
I am not sure whether you have added this property on purpose, but we should not need to use this I guess.